src/actions/topicActions.js

```diff
@@ -0,0 +1,15 @@
+import * as types from './actionTypes';
+import topicApi from '../api/topicApi';
+
+export const getTopicList = (topics) => ({
+  type: types.GET_TOPICS,
+  topics
```

Comment:
The shorthand object at the top. https://github.com/airbnb/javascript#objects--grouped-shorthand